language-server/modules/langserver-core/build.gradle

```diff
@@ -83,7 +83,6 @@ dependencies {
     testCompile 'org.awaitility:awaitility'
     testImplementation 'org.powermock:powermock-mockito-release-full'
     testImplementation 'org.powermock:powermock-module-testng-common'
-    
```

Comment:
Was this introduce in a client declaration PR?

Reply:
yes. https://github.com/ballerina-platform/ballerina-lang/pull/37584/files